LoopTests/Mock Stores/MockGlucoseStore.swift

```diff
@@ -0,0 +1,181 @@
+        })
+    }
+}
+
+extension MockGlucoseStore {
+    public var bundle: Bundle {
+        return Bundle(for: type(of: self))
+    }
+
+    public func loadFixture<T>(_ resourceName: String) -> T {
+        let path = bundle.path(forResource: resourceName, ofType: "json")!
+        return try! JSONSerialization.jsonObject(with: Data(contentsOf: URL(fileURLWithPath: path)), options: []) as! T
+    }
+    
+    var counteractionEffectToLoad: String {
+        switch testType {
+        case .flatAndStable:
+            return "flat_and_stable_counteraction_effect"
+        case .highAndStable:
+            return "high_and_stable_counteraction_effect"
+        case .highAndRisingWithCOB:
+            return "high_and_rising_with_cob_counteraction_effect"
+        case .lowAndFallingWithCOB:
+            return "low_and_falling_counteraction_effect"
+        case .lowWithLowTreatment:
+            return "low_with_low_treatment_counteraction_effect"
+        case .highAndFalling:
+            return "high_and_falling_counteraction_effect"
+        }
+    }
+    
+    var momentumEffectToLoad: String {
+        switch testType {
+        case .flatAndStable:
+            return "flat_and_stable_momentum_effect"
+        case .highAndStable:
+            return "high_and_stable_momentum_effect"
+        case .highAndRisingWithCOB:
+            return "high_and_rising_with_cob_momentum_effect"
+        case .lowAndFallingWithCOB:
+            return "low_and_falling_momentum_effect"
+        case .lowWithLowTreatment:
+            return "low_with_low_treatment_momentum_effect"
+        case .highAndFalling:
+            return "high_and_falling_momentum_effect"
+        }
+    }
+    
+    var glucoseStartDate: Date {
+        switch testType {
+        case .flatAndStable:
+            return dateFormatter.date(from: "2020-08-11T20:45:02")!
+        case .highAndStable:
+            return dateFormatter.date(from: "2020-08-12T12:39:22")!
+        case .highAndRisingWithCOB:
+            return dateFormatter.date(from: "2020-08-11T21:48:17")!
+        case .lowAndFallingWithCOB:
+            return dateFormatter.date(from: "2020-08-11T22:06:06")!
+        case .lowWithLowTreatment:
+            return dateFormatter.date(from: "2020-08-11T22:23:55")!
+        case .highAndFalling:
+            return dateFormatter.date(from: "2020-08-11T22:59:45")!
+        }
+    }
+    
+    var latestGlucoseValue: Double {
+        switch testType {
+        case .flatAndStable:
+            return 123.42849966275706
+        case .highAndStable:
+            return 200.0
+        case .highAndRisingWithCOB:
+            return 129.93174411197853
+        case .lowAndFallingWithCOB:
+            return 75.10768374646841
+        case .lowWithLowTreatment:
+            return 81.22399763523448
+        case .highAndFalling:
+            return 200.0
```

Comment:
Because the algorithm only explicitly needs the most recent cached glucose value (other cached glucose values are used to generate effects, though), I chose to just create that 'last' glucose value